exasol/nb_connector/transformers_extension_wrapper.py

```diff
@@ -72,29 +73,40 @@ def deploy_language_container(conf: Secrets,
             The language alias of the extension's language container.
     """
 
-    deployer = TeLanguageContainerDeployer.create(
-        dsn=get_optional_external_host(conf),
-        db_user=conf.get(CKey.db_user),
-        db_password=conf.get(CKey.db_password),
-        bucketfs_name=conf.get(CKey.bfs_service),
-        bucketfs_host=conf.get(CKey.bfs_host_name, conf.get(CKey.db_host_name)),
-        bucketfs_port=get_optional_bfs_port(conf),
-        bucketfs_user=conf.get(CKey.bfs_user),
-        bucketfs_password=conf.get(CKey.bfs_password),
-        bucketfs_use_https=str_to_bool(conf, CKey.bfs_encryption, True),
-        bucket=conf.get(CKey.bfs_bucket),
-        saas_url=conf.get(CKey.saas_url),
-        saas_account_id=conf.get(CKey.saas_account_id),
-        saas_database_id=conf.get(CKey.saas_database_id),
-        saas_database_name=conf.get(CKey.saas_database_name),
-        saas_token=conf.get(CKey.saas_token),
-        path_in_bucket=PATH_IN_BUCKET,
-        language_alias=language_alias,
-        use_ssl_cert_validation=str_to_bool(conf, CKey.cert_vld, True),
-        ssl_trusted_ca=conf.get(CKey.trusted_ca),
-        ssl_client_certificate=conf.get(CKey.client_cert),
-        ssl_private_key=conf.get(CKey.client_key)
-    )
+    backend = get_backend(conf)
```

Comment:
See suggestion above, if you go with the `kwargs` approach in the future you could even factor out more complex build up into functions e.g.

```python
kwargs = {
 # Default/Standard Values
}

if backend == StorageBackend.onprem:
     kwargs.update(onprem_params())
 else:
     kwargs.update(saas.params())
      
deployer = DeployerClass(**kwargs)
```

Reply:
OK, it makes sense. Thanks.